src/pynwb/file.py

```diff
@@ -109,7 +109,7 @@ class NWBFile(MultiContainerInterface):
             'get': 'get_stimulus_template'
         },
         {
-            'attr': 'modules',
```

Comment:
Does `nwb.modules['module_name']` still work?

Reply:
Yes it still works. I just added a test for it